main_net_config.conf

```diff
@@ -464,20 +578,30 @@ block = {
   proposalExpireTime = 259200000 // 3 day: 259200000(ms)
 }
 
-# Transaction reference block, default is "head", configure to "solid" can avoid TaPos error
-trx.reference.block = "solid" // head;solid;
+# Transaction reference block, default is "solid", configure to "head" may accur TaPos error
```

Comment:
Is accur anything else?

Reply:
This is a cheap mistake, it should read: incur